### PR TITLE
[17.0][MIG] account_payment_purchase: Migration to 17.0 

### PR DESCRIPTION
**Pull request by ramiadavid**

### CODE REVIEW

**Review by smcqueen81 — APPROVED** [review]
Funcional ok

File: account_payment_purchase/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+You are able to add a payment mode directly on a partner. This payment
+mode is automatically associated to the purchase order, then on related
+invoice. This default value could be change in a draft purchase or draft
+invoice. When you create a payment order, only invoices related to
+chosen payment mode are displayed. Invoices without any payment mode are
+displayed too.
+
+This module doesn't add any feature, but it is used by several other
```
Comment (pedrobaeza) [inline account_payment_purchase/readme/USAGE.md]: This paragraph is not correct.

Comment (ramiadavid) [reply account_payment_purchase/readme/USAGE.md]: updated

File: account_payment_purchase/readme/USAGE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+You are able to add a payment mode directly on a partner.
+This payment mode is automatically associated to the purchase order, then on
+related invoice.
+This default value could be change in a draft purchase or draft invoice.
+When you create a payment order, only invoices related to chosen payment mode
+are displayed.
+Invoices without any payment mode are displayed too.
+
+
+This module doesn't add any feature, but it is used by several other modules.
```
Comment (pedrobaeza) [inline account_payment_purchase/readme/USAGE.md]: I still see this text.

Comment (ramiadavid) [reply account_payment_purchase/readme/USAGE.md]: This is the text in version 16, I have not changed anything:
https://raw.githubusercontent.com/OCA/bank-payment/16.0/account_payment_purchase/readme/USAGE.rst

Comment (pedrobaeza) [reply account_payment_purchase/readme/USAGE.md]: Yes, I know, but now reviewing this migration, I have discovered that it's not correct (both in v16 nor v17), so it's the occasion to fix it, at least in v17.

Comment (ramiadavid) [reply account_payment_purchase/readme/USAGE.md]: ok, And what would be the correct text?

Comment (pedrobaeza) [reply account_payment_purchase/readme/USAGE.md]: I think the first paragraph is enough. Just remove this one.

Comment (ramiadavid) [reply account_payment_purchase/readme/USAGE.md]: that's OK?

**Review by pedrobaeza — APPROVED** [review]
Thanks

/ocabot merge nobump

**Pull request merged**